### ISSUE TITLE
Fail on ADC read on boot

### ISSUE THREAD

**Issue by lptr** [cornucopia-machines/ugly-duckling-firmware#323]
```text
waiting for download
--- Error: ClearCommError failed (PermissionError(13, 'The device does not recognize the command.', None, 22))
--- Waiting for the device to reconnect.
W (377) i2c: This driver is an old driver, please migrate your auWQ7k6R+YVoXpPDw4zkeq+96CI8I2Zpdgkq4iDun","+ojcyjQR8W5l/lTNB6FixcyevQKBgQDinaqCKvGjEUdcoHW/YCIJBbP0P7gNUwDU","4LoOLHScUAmhy+SqUJyoFjdHaW9n2xKngPxg8cxHJLLarYF3/upLnFjUNc8BeBcy","mtxX6d773SgyJdidcpm1nwAfcCNYvEkCLi1ncJM6T90DW2lijGH4KidgsIYnU/AP","XZqgOB+VHQKBgBWk9zPfLTcSiSZAqZUaKGj0SL07ykEvrNIw+KS+vQiUk9vnZfHr","msXHsc9gUIix/JLb2hSrWj/CgDlnAtrYRsFoRe+znX7pxSep5cEluVLnIsg/kR9Y","W/lbgL/JNFeu7WO7Wl6/BLqGdM67YTj03lLU32mxcdw9kgPG7xo7wdbxAoGBAMIQ","WSNNmU6XDnChLRStQ5hMXPYO1xP5/91NcaePpHD0DbuT+wdrLyy2kbJC9VeQAOKX","EAoJJ3HJWKzyJsxW7dfAVUWtqs9MQzlf6tAShJVVqAWB6YWzzTP+rME/a9WisDcA","ynspqbsnB1LHpnDxXTiDCOPBZ7EoLY1LEI6iYVF1AoGAHVxSh54DPENGbrwoDv+z","0iKJFobooppHVortFd7ZUhp7JpTFNb0QgmBAwQH38QpLy+pG5TOInRj3KozjMHc4","r8Lmdqg1wHerruFLb5ixndybfKLcwz/aGc0nIqczasi9LGsXUl2V5UlCu4tONpU+","zQvSU6HjU6VYtzReIR/V57E=","-----END PRIVATE KEY-----"]}
I (889) farmhub: Initializing LED driver on pin STATUS2
D (893) farmhub: Creating task status-2 with priority 1 and stack size 2048
I (900) farmhub: Initializing DRV8833 on pins ain1 = AIN1, ain2 = AIN2, bin1 = BIN1, bin2 = BIN2, fault = NFault, sleep = C4
W (911) farmhub:pm: Light sleep is disabled in debug mode
D (917) farmhub:wifi: Registering WiFi handlers
W (923) vfs: esp_vfs_register_fd_range is successful for range <54; 64) and VFS ID 4
D (951) farmhub: Creating task wifi-driver with priority 1 and stack size 4096
D (952) farmhub: Creating task mdns:init with priority 1 and stack size 4096
I (957) farmhub:mdns: initializing
D (958) farmhub: Creating task ntp-sync with priority 1 and stack size 4096
D (962) farmhub:wifi: Connecting using stored credentials to aardwark (password '3Ant5Eater8')
D (974) farmhub: Creating task mqtt with priority 1 and stack size 5120
D (980) farmhub:wifi: Stopping WiFi (was up 0 ms)
D (987) farmhub: Creating task mqtt:incoming with priority 1 and stack size 4096
D (988) farmhub:mqtt: server: motherhen.kertkaland.com:8883, client ID is 'ugly-duckling-chicken-door'
D (993) farmhub:wifi: Starting station
D (1002) farmhub: Creating task switch-manager with priority 1 and stack size 3072
D (1057) farmhub:wifi: Started
I (1061) farmhub: Initializing FarmHub kernel version 0.32.0-rc-12-49-g75bf563 on mk6 instance 'chicken-door' with hostname 'chicken-door' and MAC address 34:85:18:50:56:fc
D (1076) farmhub: Creating task status-update with priority 1 and stack size 3072
D (1093) farmhub: Kernel state changed from 0 to 1
I (1144) farmhub: Initializing analog battery driver on pin BATTERY
D (1159) farmhub: Creating task console with priority 1 and stack size 3072
ESP_ERROR_CHECK failed: esp_err_t 0x107 (ESP_ERR_TIMEOUT) at 0x4200d221
--- 0x4200d221: farmhub::kernel::AnalogPin::analogRead() const at D:/Workspace/kivancsi/ugly-duckling/main/kernel/Pin.hpp:159 (discriminator 2)

file: "./main/kernel/Pin.hpp" line 159
func: int farmhub::kernel::AnalogPin::analogRead() const
expression: adc_oneshot_read(handle, channel, &value)
```

Commit: 75bf563fa7b12526cf6bc641ea1cecd275e8dbd6

**Comment by lptr** [comment]
Status led stays on. Weird.

**Comment by lptr** [comment]
This is fixed via #325, though it's unclear why we seem to have this only with `chicken-door`...
